src/views/Order.vue

```diff
@@ -346,16 +346,14 @@ export default defineComponent({
       let originFacilityName = "", resp;
 
       try {
-        resp = await OrderService.getRerouteOrderBrokeringHistory({ "token": this.token })
+        resp = await OrderService.getRerouteOrderFacilityChangeHistory({ "token": this.token, facilityId: "PICKUP_REJECTED" })
 
-        if(!hasError(resp) && resp.data?.brokeringHistory.length) {
-          const oldestBrokeringHistory = resp.data.brokeringHistory.reduce((oldest: any, current: any) => {
-              return current.changeDatetime < oldest.changeDatetime ? current : oldest;
```

Comment:
We need to still keep this logic. Let me know if I have missed something here.

@dt2patel Please confirm here once that whether we will be showing the recent facility that rejected the order or the facility from where the order was first rejected. Just for info, in the custom app we are showing the recent facility that rejected the order.